lambda/lambda-metering-users.py

```diff
@@ -0,0 +1,52 @@
+import base64
+import gzip
+import json
+import os
+from botocore.exceptions import ClientError
+import boto3
+import logging
+from datetime import datetime
+
+def lambda_handler(event, context):
+    cw_data = event['awslogs']['data']
+    compressed_payload = base64.b64decode(cw_data)
+    uncompressed_payload = gzip.decompress(compressed_payload)
+    data = json.loads(uncompressed_payload.decode('utf-8'))
+    sqs_url = os.environ.get("SqsMeteringUsersUrl")
+    customerID = os.environ.get("customerID")
+    logging.getLogger().setLevel(logging.ERROR)
```

Comment:
Move this line out of the function to the top-level of the script.

In general, Python loggers should only be created and configured at the top-level of a module. They should not be created inside a function because Python loggers are not properly garbage collected, so might be causing memory leak when created in a repeatedly called function. Even though it's probably not the case with this Lambda, we should use the best practice.